src/lsst/cmservice/handlers/element_handler.py

```diff
@@ -392,6 +392,16 @@ async def check(
                 await element.update_values(session, status=status)
                 return (changed, status)
 
+        if element.level != LevelEnum.job:
+            jobs = await element.get_jobs(session, remaining_only=True)
+        else:
+            jobs = []
+        for job_ in jobs:
+            if job_.status.value <= StatusEnum.accepted.value:
+                status = StatusEnum.running  # FIXME
```

Comment:
Well, you begged the question...  What _should_ happen here?  Maybe add a little explanation to help whomever will eventually have to fix it (even future you!)

Reply:
Don't know yet.   Depends on what we come up with in terms of how we want to flag things, pause jobs etc.  
I will helpfully change this to   `# FIXME, revisit someday`